### ISSUE TITLE
TLA detection via ecmascript package not fixed in Meteor 3.0.4 because of old dependencies in published package

### ISSUE THREAD

**Issue by perbergland** [meteor/meteor#13424]
Something is still broken in the exact same way as before - is the correct version of reify really used?

Adding "await 3" to tla-file.js makes the bug go away.

SImple test repro - please test with this repo before closing the bug

https://github.com/perbergland/tla-repro/tree/main

**Comment by perbergland** [comment]
I checked and the published package(s) have the wrong reify version.

Here are all the reify versions found in the downloaded ecmascript 0.16.9 (macos, Apple Silicon):

```
find ~/.meteor/packages/ecmascript/0.16.9/ -type f -path "*/@meteorjs/reify/package.json" -exec sh -c 'echo "File: $1"; jq "{name,version}" < "$1"' _ {} \;

File: /Users/per.bergland/.meteor/packages/ecmascript/0.16.9//plugin.compile-ecmascript.os/npm/node_modules/meteor/babel-compiler/node_modules/@meteorjs/reify/package.json
{
  "name": "@meteorjs/reify",
  "version": "0.25.1"
}
File: /Users/per.bergland/.meteor/packages/ecmascript/0.16.9//plugin.compile-ecmascript.os/npm/node_modules/meteor/modules/node_modules/@meteorjs/reify/package.json
{
  "name": "@meteorjs/reify",
  "version": "0.24.0"
}

```

**Comment by perbergland** [comment]
Similar problem with @meteorjs/babel:

```
find ~/.meteor/packages/ecmascript/0.16.9/ -type f -path "*/@meteorjs/babel/package.json" -exec sh -c 'echo "File: $1"; jq "{name,version}" < "$1"' _ {} \;

File: /Users/per.bergland/.meteor/packages/ecmascript/0.16.9//plugin.compile-ecmascript.os/npm/node_modules/meteor/babel-compiler/node_modules/@meteorjs/babel/package.json
{
  "name": "@meteorjs/babel",
  "version": "7.20.0-beta.4"
}
```

**Comment by perbergland** [comment]
If I understand the publish process for compilers correctly, this can be fixed by publishing a new version of ecmascript that bundles the 7.11.1 version of babel-compiler instead of whatever was used when publishing 0.16.9

**Comment by perbergland** [comment]
Related to this, I updated refapp:meteor-typescript to use the new reify by publishing a new version with 3.0.3 dependencies in the packages.

That then made the package use the new reify 0.25.3 and now the TLA detection flags the cases it didn’t with the old version.

Unfortunately, this exposed some other issues that I will file in the reify github repo

**Comment by perbergland** [comment]
Fixed in 3.1
